source/css/main.css.scss

```diff
@@ -0,0 +1,5 @@
+@charset "utf-8";
+
+@import
+        "_normalize.css.scss"
```

Comment:
Imported partial `_normalize.css.scss` should be written as `normalize.css`<br>Line should be indented 0 spaces, but was indented 8 spaces<br>Declaration should not have a space before the terminating semicolon